modules/end-user-guide/partials/proc_adding-vs-code-extensions.adoc

```diff
@@ -0,0 +1,47 @@
+
+[id="adding-vs-code-extensions"]
+= Adding `VS Code` extensions
+
+To have the Microsoft Visual Studio Code - Open Source IDE automatically install chosen extensions, you have to add an `extensions.json` file to the remote Git repository that contains your project source code and that will be cloned into workspaces.
+
+.Prerequisites
+* {prod-short} has network access to the link:https://www.open-vsx.org/[OpenVSX registry].
++
+[TIP]
+====
+To install recommended extensions in a restricted environment, consider these options:
+
+* xref:administration-guide:configuring-the-open-vsx-registry-url.adoc[] to point to your OpenVSX registry.
+
+* xref:defining-a-common-ide.adoc[].
+
+* link:https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix[Installing extensions from VSX files].
+====
+
+.Procedure
+
+. Create a `.vscode/extensions.json` file in the remote Git repository.
+
+. Go to the link:https://www.open-vsx.org/[Open VSX registry] and copy the **Unique identifier** of each chosen extension from the extension-listing page to the `extensions.json` file as follows:
```

Comment:
Is this the latest version of vsx page design?

Reply:
Thank you so much for catching this!
Since I tested this procedure, the html page template on the VSX website has been updated and no longer features the info that I tested this with.
I've checked the new VSX pages and have written a new commit to update this.
Thank you again.